### PR TITLE
feat: add design system page

### PR DESCRIPTION
**Pull request by Mythie**
Adds a page for the design system that we have created in Figma. Can be accessed on the marketing site by navigating to `/design-system`

### CODE REVIEW

File: apps/marketing/content/design-system.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: Design System
+---
+
+# Design System
+
+At Documenso we aim to be a design driven company. We believe that design is not just about how things look, but also how they work. We want to make sure that our products are easy to use and intuitive. We also want to make sure that our products are consistent and that they look and feel like they belong together.
+
+To achieve this we have created our design system which contains our primitives, token and screens alongside branding assets.
+
+We're open-sourcing our design system so that you can see how we build our products and think about design as a whole.
```
Comment (Mythie) [inline apps/marketing/content/design-system.mdx:11]: @fmerian @ElTimuro Do your thang 💅🏻

**Comment by adithyaakrishna** [comment]
These three orgs have been my source for inspiration wrt to design systems, 

- Airbnb, https://airbnb.design/
- Spotify, https://spotify.design/
- Atlassian, https://atlassian.design/

File: apps/marketing/content/design-system.mdx (hunk 1 above)
Comment (dephraiim) [reply apps/marketing/content/design-system.mdx:11]: Looks like its above your paygrade

Comment (Mythie) [reply apps/marketing/content/design-system.mdx:11]: no doubt no doubt

File: apps/marketing/content/design-system.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+<iframe
+  src="https://documen.so/design-system-embed"
```
Comment (fmerian) [inline apps/marketing/content/design-system.mdx:14]: what do we want to embed here? 

we plan to publish the design system to the Figma community so anyone can remix it. the current embed links to a read-only page. it'd be best to link to the Figma community file.

[See example](https://www.figma.com/community/file/1182079839084966604/ADS%3A-Design-Tokens)

File: apps/marketing/content/design-system.mdx (hunk 1 above)
Comment (fmerian) [reply apps/marketing/content/design-system.mdx:11]: `/design` may be a better (simpler) URI?

Comment (adithyaakrishna) [reply apps/marketing/content/design-system.mdx]: Yes, I love the idea! 

Just imagine, https://design.documen.so, sounds powerful 😮‍💨

Comment (fmerian) [reply apps/marketing/content/design-system.mdx]: for SEO, it's best to keep it under the main `documenso.com` domain, e.g. `documenso.com/design` (although [design.documenso.com](https://design.documenso.com/)) 🤯

File: apps/marketing/content/design-system.mdx (hunk 2 above)
Comment (Mythie) [reply apps/marketing/content/design-system.mdx:16]: Honestly an embed vs the figma community file is of very little difference. Jumping into a read-only will still let you create a local copy which is all the same.

This provides some interactivity on the page which is quite low-touch right now.

Comment (fmerian) [reply apps/marketing/content/design-system.mdx:16]: > Honestly an embed vs the figma community file is of very little difference. Jumping into a read-only will still let you create a local copy which is all the same.

nope, there are *not so little* differences. 
1. when in read-only mode, you can just view the file. you don't access the data you need -- text style, spacings, measurements, and so on. 
2. you can't duplicate a Figma file or save it to your drafts. you have to invite people manually. plus, 
3. it doesn't leverage Figma as a distribution channel -- no likes, no comments, no search, no discovery.

these are significant differences. it's best to publish it to the Figma community and add a link here :)

Comment (ElTimuro) [reply apps/marketing/content/design-system.mdx:16]: "when in read-only mode, you can just view the file. you don't access the data you need -- text style, spacings, measurements, and so on." => you can access those, just need to be logged in, I checkd

Comment (ElTimuro) [reply apps/marketing/content/design-system.mdx:16]: - nur sure about 2, 3 is a thing
- Why wouldn't we publish after embed? Leverage both

Comment (fmerian) [reply apps/marketing/content/design-system.mdx:16]: > Why wouldn't we publish after embed?
compound effect

**Pull request closed without merging**